File: sentimental-mario-less/mario.py
from cs50 import get_int

# Asks user for height while rejecting values less than 1 and greater than 8
while True:
    height = get_int("Height: ")
    if 0 < height <= 8:
        break

# Prints the blocks
for i in range(height):

    # Prints spaces to right align
    if height > 1:
        for j in range(height, i + 1, -1):
            print(" ", end="")

    # Prints hashes
    for k in range(i + 1):
        print("#", end="")

    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Asks user for height while rejecting values less than 1 and greater than 8
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    // Prints the blocks
    for (int i = 0; i < height; i++)
    {
        // Prints spaces to right align
        if (height > 1)
        {
            for (int j = height; j > i + 1 ; j--)
            {
                printf(" ");
            }
        }

        // Prints hashes
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        printf("\n");
    }
}